src/contracts/ProposalPayloadAaveEcosystemReserveWithVoting.sol

```diff
@@ -22,17 +21,18 @@ contract ProposalPayloadAaveEcosystemReserveWithVoting {
       0x25F2226B597E8F9514B3F68F00f494cF4f286491
     );
 
-  constructor(address aaveEcosystemReserveV2Impl, uint256 proposalId) {
```

Comment:
Why not keeping it this way?

Reply:
Because at deployment time of this payload, we will not know the proposal Id of LongExecutor. So we need to pass it as arg on the execute method (when we will know the id)